spec/jobs/qualification/expiration_mailer_job_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+require 'spec_helper'
+
+describe Qualification::ExpirationMailerJob do
+  before { ActionMailer::Base.deliveries.clear }
+
+  subject(:job) { described_class.new }
+  subject(:qualification) do
+    load './db/seeds/custom_contents.rb'
```

Comment:
das besser in einem before block

Reply:
subject in before block geht nicht

Reply:
nein ... ohne das subject ... einfach das laden der custom contents

```
before { load './db/seeds/custom_contents.rb' }
```